helix/principles/architecture-principles/modules.rst

```diff
@@ -62,6 +62,11 @@ should always be in the domain language. (See :doc:`/principles/architecture-pri
     The Social feature enables Twitter feeds on the pages as well as
     allows the editor to add Facebook metadata to the website pages.
 
+  Feature/Cart
```

Comment:
This isn't actually in Habitat, is it?

Reply:
This comes from Sitecore.HabitatHome.Commerce: https://github.com/Sitecore/Sitecore.HabitatHome.Commerce

Reply:
Ah ha. this admonition is getting updated in the Helix Examples branch too. Maybe we need a separate admonition section below this w/ an example list of modules from Habitat Home Commerce.